gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -338,6 +338,16 @@ gpii.settingsHandlers.writeFile = function (content, options) {
 gpii.settingsHandlers.handleFileSolutionEntry = function (solutionEntry, modifier, parser, isWrite) {
     var options = solutionEntry.options;
 
+    var initialContent = options.initialContent;
+    var createFile = options.createFile;
+    //create file, if file does not exist, and set initial content of the file
+    if (createFile) {
+        var exists = fs.existsSync(options.filename);
+        if (!exists) {
+            fs.writeFileSync(options.filename, initialContent, options.encoding || "utf-8");
```

Comment:
Hi,
I have remove the initialContent. Previously it crashes if the file is empty but now it's working so I don't think we need that.
I've made the changes and used the code in gpii.settingsHandlers.writeFile utility.
https://github.com/GPII/universal/pull/374
I'm going to separate it from this pull request.